### PR TITLE
feat: add google analytics

### PR DESCRIPTION
**Pull request by Enter-tainer**
thanks @SukkaW

### CODE REVIEW

File: src/html.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ export default function HTML (props) {
           content="width=device-width, initial-scale=1, shrink-to-fit=no"
         />
         <script src="/script.js"></script>
+        <script dangerouslySetInnerHTML={{
+          __html: `window.ga_tid = "UA-124485594-1";
+          window.ga_api = "https://margatroid.xyz/vue.min.js";`,
```
Comment (CBW2007) [inline src/html.js:17]: 403🤔为啥不用官方的

Comment (CBW2007) [reply src/html.js:17]: 如果是特殊的话当我没说（

Comment (Enter-tainer) [reply src/html.js:17]: see https://github.com/SukkaW/cloudflare-workers-async-google-analytics#say-hello-to-easylist

Comment (CBW2007) [reply src/html.js:17]: 👌

**Comment by CBW2007** [comment]
只是顶掉request_changes而已（

**Pull request merged**